<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QXNConfigDialog</class>
 <widget class="QDialog" name="QXNConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QXNeur settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/config/general.png</normalon>
       </iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Layout</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/config/layout.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Keyboard shortcuts</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/config/shortcuts.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Applications</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/config/applications.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sounds</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/config/sound.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Abbreviations</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/config/abbreviations.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Appearance</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/config/appearance.png</normalon>
       </iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="configurationPagesStack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="generalTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="generalTab">
          <attribute name="title">
           <string>General</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="manualModeCheckBox">
             <property name="text">
              <string>Work in manual mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="selfEducateCheckBox">
             <property name="text">
              <string>Self educate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkLanguageDuringInputCheckBox">
             <property name="text">
              <string>Check language during input</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="saveSelectionCheckBox">
             <property name="text">
              <string>Save selection after layout switching</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="logInputCheckBox">
             <property name="text">
              <string>Log the keyboard input</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="typographyTab">
          <attribute name="title">
           <string>Typography</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="correctAccidentalCapsLockCheckBox">
             <property name="text">
              <string>Correct aCCIDENTAL CapsLock</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="correctTwoCapitalLettersCheckBox">
             <property name="text">
              <string>Correct two CApital letters</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="correctSpaceWithPunctuationCheckBox">
             <property name="text">
              <string>Correct space with punctuation</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="advancedTab">
          <attribute name="title">
           <string>Advanced</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="dontProcessOnEnterCheckBox">
             <property name="text">
              <string>Don't process word on Enter or Tab</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="flushBuffersOnEnterCheckBox">
             <property name="text">
              <string>Flush internal buffers after Enter or Tab</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="miscOptionsGroupBox">
             <property name="title">
              <string>Misc. options</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="eventSendDelayLabel">
                <property name="text">
                 <string>Event send delay (ms.):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="eventSendDelaySpinBox">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>50</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="logLevelLabel">
                <property name="text">
                 <string>Logging level:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="logLevelCombo">
                <item>
                 <property name="text">
                  <string>Error</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Warning</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Log</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Debug</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Trace</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutPage">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>53</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Layout</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="shortcutsPage">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Keyboard shortcuts</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="applicationsPage">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="rememberLayoutCheckBox">
         <property name="text">
          <string>Remember layout for every window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="customApplicationsGroupBox">
         <property name="title">
          <string>Custom applications</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="customApplicationsList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="addApplicationButton">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/actions/add.png</normaloff>:/actions/add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeApplicationButton">
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/actions/remove.png</normaloff>:/actions/remove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>76</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="editApplicationButton">
              <property name="text">
               <string>Edit...</string>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/actions/edit.png</normaloff>:/actions/edit.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="soundsPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="soundsGroupBox">
         <property name="title">
          <string>Enable sounds</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" colspan="2">
           <widget class="QTableWidget" name="soundsTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::DotLine</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>XneurStart</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>XneurReload</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>XneurStop</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>PressKeyLayout1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>PressKeyLayout2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>PressKeyLayout3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>PressKeyLayout4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>EnableLayout1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>EnableLayout2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>EnableLayout3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>EnableLayout4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>AutomaticChangeWord</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>ManualChangeWord</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>ChangeString</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>ChangeSelected</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TranslitSelected</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>ChangecaseSelected</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>ReplaceAbbreviation</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>CorrectIncidentalCaps</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>CorrectTwoCapitalLetter</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>ExecuteUserAction</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Action</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Sound file</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>XNeur start</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string/>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>XNeur reload</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>XNeur stop</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>Layout 1 key press</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>Layout 2 key press</string>
             </property>
            </item>
            <item row="5" column="0">
             <property name="text">
              <string>Layout 3 key press</string>
             </property>
            </item>
            <item row="6" column="0">
             <property name="text">
              <string>Layout 4 key press</string>
             </property>
            </item>
            <item row="7" column="0">
             <property name="text">
              <string>Layout 1 activated</string>
             </property>
            </item>
            <item row="8" column="0">
             <property name="text">
              <string>Layout 2 activated</string>
             </property>
            </item>
            <item row="9" column="0">
             <property name="text">
              <string>Layout 3 activated</string>
             </property>
            </item>
            <item row="10" column="0">
             <property name="text">
              <string>Layout 4 activated</string>
             </property>
            </item>
            <item row="11" column="0">
             <property name="text">
              <string>Automatic word convert</string>
             </property>
            </item>
            <item row="12" column="0">
             <property name="text">
              <string>Manual word convert</string>
             </property>
            </item>
            <item row="13" column="0">
             <property name="text">
              <string>Convert last string</string>
             </property>
            </item>
            <item row="14" column="0">
             <property name="text">
              <string>Convert selection</string>
             </property>
            </item>
            <item row="15" column="0">
             <property name="text">
              <string>Transliterate selection</string>
             </property>
            </item>
            <item row="16" column="0">
             <property name="text">
              <string>Change selection case</string>
             </property>
            </item>
            <item row="17" column="0">
             <property name="text">
              <string>Replace abbreviation</string>
             </property>
            </item>
            <item row="18" column="0">
             <property name="text">
              <string>Correct aCCIDENTIAL CapsLock</string>
             </property>
            </item>
            <item row="19" column="0">
             <property name="text">
              <string>Correct TWo capital letters</string>
             </property>
            </item>
            <item row="20" column="0">
             <property name="text">
              <string>Execute user action</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="editSoundButton">
            <property name="text">
             <string>Edit...</string>
            </property>
            <property name="icon">
             <iconset resource="../../images/images.qrc">
              <normaloff>:/actions/edit.png</normaloff>:/actions/edit.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>243</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="abbreviationsPage">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="ignoreLayoutForAbbreviationsCheckBox">
         <property name="text">
          <string>Ignore layout for abbreviations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="abbreviationTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DotLine</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Abbreviation</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Full text</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addAbbreviationButton">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../../images/images.qrc">
             <normaloff>:/actions/add.png</normaloff>:/actions/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeAbbreviationButton">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="../../images/images.qrc">
             <normaloff>:/actions/remove.png</normaloff>:/actions/remove.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="editAbbreviationButton">
           <property name="text">
            <string>Edit...</string>
           </property>
           <property name="icon">
            <iconset resource="../../images/images.qrc">
             <normaloff>:/actions/edit.png</normaloff>:/actions/edit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="appearancePage"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>manualModeCheckBox</tabstop>
  <tabstop>selfEducateCheckBox</tabstop>
  <tabstop>checkLanguageDuringInputCheckBox</tabstop>
  <tabstop>logInputCheckBox</tabstop>
  <tabstop>correctAccidentalCapsLockCheckBox</tabstop>
  <tabstop>correctTwoCapitalLettersCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QXNConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QXNConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>configurationPagesStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
